specification/quota/resource-manager/Microsoft.Quota/preview/2021-03-15-preview/examples/putMachineLearningServicesQuotaRequestLowPriority.json

```diff
@@ -29,9 +33,7 @@
     },
     "200": {
       "headers": {
-        "location": "https://management.azure.com/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605?api-version=2021-03-15-preview",
-        "Retry-After": "30",
-        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/operationsStatus/2B5C8515-37D8-4B6A-879B-CD641A2CF605?api-version=2021-03-15-preview"
+        "location": "https://management.azure.com/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605?api-version=2021-03-15-preview"
```

Comment:
Why this one return `location` header instead of `azure-asyncoperation`?

Reply:
Based on - https://github.com/Azure/azure-sdk-for-net/issues/3020
if Azure-AsyncOperation was polled and the operation succeeded then the Location header should be retrieved (if possible) in order to get the correct response code.
For 202 Azure-AsyncOperation is checked.
When the operation is completed, then 200 is returned with Location header.